husky_components/datagrid/main.js

```diff
@@ -2311,6 +2324,11 @@
                             searchFields: this.options.searchFields.join(',')
                         });
                     }
+                    if (searchString !== '') {
+                        this.storage.set('url', url);
+                    } else {
+                        this.storage.remove('url');
```

Comment:
the url will always be saved when the sorting/search is changed therefor all the parameters will be saved.